xtra-libp2p-offer/src/deprecated.rs

```diff
@@ -0,0 +1,4 @@
+pub mod maker;
+pub mod protocol;
+
+pub const PROTOCOL: &str = "/itchysats/offer/1.0.0";
```

Comment:
❓ Why is this called `deprecated`, when for `rollover` we added versions, i.e. `v_1_0_0` and `v_2_0_0` - I think it would help the code base to have some consistency here.

Reply:
Because it's much clearer (I introduced the original horrible terminology). I've adapted the rollover one in a separate PR).

Reply:
Correction: I forgot to actually open the PR 5 days ago! [Here it is](https://github.com/itchysats/itchysats/pull/2706).